src/main/java/Task.java

```diff
@@ -0,0 +1,20 @@
+public class Task {
+    protected String description;
+    protected boolean isDone;
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+    public String getStatusIcon() {
+        return (isDone ? "X" : " "); // mark done task with X
+    }
+    public String getDescription() {
+        return this.description;
+    }
+    public void markAsDone() {
```

Comment:
I like how you separated the mark done and undone into 2 methods

Reply:
Thank you for your validating my good practices.